### PR TITLE
Update back-end-server.md

### PR DESCRIPTION
**Pull request by dariomws306**
SQL 2017 is supported on SfB Server 2015 so it would be supported on Mirroring:
https://docs.microsoft.com/skypeforbusiness/plan-your-deployment/requirements-for-your-environment/server-requirements#DBs

AlwaysOn is supported on SfB Server 2019, Mirroring not:
https://techcommunity.microsoft.com/t5/Skype-for-Business-Ignite-Blog/Ignite-2018-BRK3229-Everything-you-need-to-know-about-Skype-for/ba-p/264448

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/back-end-server.md [hunk 1]
```diff
@@ -95,9 +97,31 @@ If both the principal and the mirror Back End Servers fail, or if one of those s
...
+
+- SQL Server 2016 Standard Edition with limitations, see note below
+
+- SQL Server 2014 Enterprise Edition
+    
+- SQL Server 2012 SP2 and CU2 Enterprise Edition
+
+> [!NOTE]
+> SQL Server 2017 and SQL Server 2016 are the only versions supported by Skype for Business Server 2019.
+
```
Comment (nenonix) [inline Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/back-end-server.md]:
```suggestion
Always On Availability Groups is **not** supported in SQL 2016 and 2017 Standard Editions but you can use Always On Failover Cluster Instances. See [Editions and supported features of SQL Server 2016](https://docs.microsoft.com/sql/sql-server/editions-and-components-of-sql-server-2016?view=sql-server-2017) to learn more.
```

File: Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/back-end-server.md [hunk 2]
```diff
@@ -95,9 +97,31 @@ If both the principal and the mirror Back End Servers fail, or if one of those s
...
+> [!NOTE]
```
Comment (nenonix) [inline Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/back-end-server.md]:
```suggestion

```

File: Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/back-end-server.md [hunk 3]
```diff
@@ -95,9 +97,31 @@ If both the principal and the mirror Back End Servers fail, or if one of those s
...
+> SQL 2016 and 2017 Standard Edition limitation: Always On availability groups is not supported, you can only choose Always On failover cluster instances.
```
Comment (nenonix) [inline Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/back-end-server.md]:
```suggestion
> Always On Availability Groups is **not** supported in SQL 2016 and 2017 Standard Editions but you can use Always On Failover Cluster Instances. See [Editions and supported features of SQL Server 2016](https://docs.microsoft.com/sql/sql-server/editions-and-components-of-sql-server-2016?view=sql-server-2017) to learn more.
```

File: Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/back-end-server.md [hunk 4]
```diff
@@ -95,9 +97,31 @@ If both the principal and the mirror Back End Servers fail, or if one of those s
...
+> Refer to below link:
```
Comment (nenonix) [inline Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/back-end-server.md]:
```suggestion

```

File: Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/back-end-server.md [hunk 5]
```diff
@@ -95,9 +97,31 @@ If both the principal and the mirror Back End Servers fail, or if one of those s
...
+> https://docs.microsoft.com/en-us/sql/sql-server/editions-and-components-of-sql-server-2016?view=sql-server-2017
```
Comment (nenonix) [inline Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/back-end-server.md]:
```suggestion

```

**Comment by scanum** [comment]
@MicrosoftHeidi Hi Heidi, Changes look correct, could you please review and merge? Thanks!

**Comment by scanum** [comment]
@SerdarSoysal @MicrosoftHeidi Hi Heidi, any updates on this? Thanks

**Comment by get-itips** [comment]
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/1351

**Pull request merged**